lib/dashboard-widgets/WP_Auth0_Dashboard_Plugins_Age.php

```diff
@@ -1,12 +1,25 @@
 <?php
-// TODO: Deprecate
+/**
+ * Class WP_Auth0_Dashboard_Plugins_Age
+ *
+ * @deprecated 3.6.0 - Not supporting dashboard widgets
```

Comment:
do you mean you no longer support dashboard widgets? In that case, I'd rephrase every mention of this to:

"Since x.y.z this library no longer supports dashboard widgets"